assets/js/store.js

```diff
@@ -61,7 +61,13 @@ Store.prototype.has = function (key/*: string*/)/*: bool*/ {
 };
 
 Store.prototype.able = function ()/*: bool*/ {
-    return !!(window.localStorage);
+    try {
+        return !!(window.localStorage);
+    }
+    catch(e) {
+        console.warn('Cookies are disabled. Thus, unable to access localStorage.');
```

Comment:
Is it possible to only handle the SecurityException? e.g. something like https://stackoverflow.com/questions/1433558/handling-specific-errors-in-javascript-think-exceptions